File: programmers/python/Lv0/주사위게임2.py
def solution(a, b, c):
    d = int(a==b) + int(b==c) + int(c==a)
    if d == 0:
        return a+b+c
    elif d == 1:
        return (a+b+c)*(a**2+b**2+c**2)
    elif d == 3:
        return (a+b+c)*(a**2+b**2+c**2)*(a**3+b**3+c**3)

File: programmers/java/Lv0/주사위게임2.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        int cnt = 0;
        if (a==b) {
            cnt++;
        }
        if (b==c) {
            cnt++;
        }
        if (c==a) {
            cnt++;
        }
        
        if (cnt==0) {
            answer = a+b+c;
        }
        else if (cnt==1) {
            answer = (a+b+c)*(a*a+b*b+c*c);
        }
        else if (cnt==3) {
            answer = (a+b+c)*(a*a+b*b+c*c)*(a*a*a+b*b*b+c*c*c);
        }
        return answer;
    }
}